src/lib/pages/validator-details/components/performance/RecentBlocksLegends.tsx

```diff
@@ -34,31 +37,38 @@ const LegendItem = ({
         {value}
       </Text>
       <Text variant="body3" color="text.dark">
-        {percent}
+        {formatRatio(ratio)}
```

Comment:
not sure if design want this part to have a fixed dp. if so change to `formatPrettyPercent`